src/RiotAPILibraries.ts

```diff
@@ -129,9 +129,21 @@ export default class RiotAPILibraries {
         message.channel.send(reply);
     }
 
-    private async getListForLanguage(message: Discord.Message, language: string) {
+    private async getListForLanguage(message: Discord.Message, language: string): Promise<void> {
         const response = await fetch(this.settings.riotApiLibraries.baseURL + language);
+
         if (response.status !== 200) {
+            const aliases = this.settings.riotApiLibraries.aliases;
+            const keys = Object.keys(aliases);
+
+            for (const index in keys) {
```

Comment:
`for (const [, key] in Object.entries(aliases))`

Reply:
It doesnt like that 🤔 

`The left-hand side of a 'for...in' statement cannot be a destructuring pattern.`

Reply:
I cant get VSC to accept Object.entries in es6, so we would need to update to es2017? 🤔

Reply:
That is fine